src/css/App.css

```diff
@@ -45,4 +45,8 @@ main {
 
 .d-flex {
   display: flex;
+}
+
+body {
+  zoom: 90%;
```

Comment:
I think that we should resize the individual items that look too big, rather than zooming out the entire app.

Reply:
It feels like everything looks too big. This is just temporary but we can scratch that change if not everyone is onboard.